src/app-layer-htp.c

```diff
@@ -2596,18 +2596,19 @@ static void HTPConfigParseParameters(HTPCfgRec *cfg_prec, ConfNode *s,
                            pval->val);
 
                 /* IPV6 or IPV4? */
+                bool is_duplicate;
                 if (strchr(pval->val, ':') != NULL) {
                     SCLogDebug("LIBHTP adding ipv6 server %s at %s: %p",
                                s->name, pval->val, cfg_prec->cfg);
-                    if (SCRadixAddKeyIPV6String(pval->val, tree, cfg_prec) == NULL) {
+                    if (!SCRadixAddKeyIPV6String(pval->val, tree, cfg_prec, &is_duplicate)) {
```

Comment:
what happens in this case? Do we leak memory?

Reply:
No because `cfg_prec` represents the entire libhtp configuration collector.